### ISSUE TITLE
Rituals Stop Working

### ISSUE THREAD

**Issue by NillerMedDild** [HellFirePvP/AstralSorcery#1070]
Hey o/

It looks like update `astralsorcery-1.12.2-1.10.12` has a bug that prevents Rituals from running. Newly built rituals are sometimes able to function for a few minutes, and then they stop. So far the issue has been confirmed with Lucerna and Aevitas.

Original issue: https://github.com/NillerMedDild/Enigmatica2Expert/issues/1082

Please let me know if you need additional information.

**Comment by Doomgull** [comment]
Long thread with lots of changes.

Can you summarize the following:
Is sponge on the server?
Are the rituals being enhanced?
What does the enhancement structure look like (screenshot) and does the enhancing crystal match the ritual?

**Comment by MaxNeedsSnacks** [comment]
1) On my server? Nope, no sponge, that doesn't even wanna launch due to other compat issues.
2) Nope, it doesn't even give me the opportunity to enhance it (with a max crystal, mind you)
3) I'm gonna have to launch up my instance of E2E for this one, but unless you guys changed sth about the ritual setup it should be a valid multiblock.

Gonna get you a screenshot ready in a bit, I wanna install Debian on my other drive first ^^

**Comment by HellFirePvP** [comment]
A few things:

1. Status effect rituals such as vicio and lucerna require at least 1 effect-tick to go off per tick to do their effect. Getting that one consistently might require a good amount of ritual acceleration, so i'll be changing that to allow for a more consistent effect without needing a lot of acceleration.
2. Rituals do get weaker during daylight, as expected. So if an unaccelerated slows down to almost a complete halt during daytime does not mean it doesn't function anymore.

That being said, there are some reports suggesting that some tiles requiring a multiblock structure invalidate that structure eventhough there are no changes. As i'm still investigating that issue and cannot find a clear reason for it, i can't say more to that than it _could_ be linked.

**Comment by Meryl** [comment]
> Status effect rituals such as vicio and lucerna require at least 1 effect-tick to go off per tick to do their effect. Getting that one consistently might require a good amount of ritual acceleration, so i'll be changing that to allow for a more consistent effect without needing a lot of acceleration.

Being the one that reported issues with his Lucerna ritual one the original issue: Has the chance of an effect-tick going off gone down? Initially it was a ritual without any form of enhancement. That worked for weeks, as in zero mob spawns. It seems unlikely to me that that was a fluke?

I have now (as in, just now) enhanced the ritual and it seems to be working now, so maybe my report (can't speak of the others) is unrelated.

**Comment by MaxNeedsSnacks** [comment]
I'm just gonna assume it's roughly the same for me then. I never tried enhancing the ritual before because the standard range was just enough and it curiously enough didn't reflect any light that I could focus back onto it with a lens anyways

**Comment by HellFirePvP** [comment]
Seems resolved for now, reopen if it becomes relevant again.

**Comment by NopeDK** [comment]
I am currently experiencing the same issue. Everything worked at first but an unknown amount of time later, maybe due to server restart, the ritual stops functioning.
Replacing the pedestal and adding the crystal gets it working again, only changing the crystal does not.

Some info:
SpongeForge 2838-7.1.8
DD&SS 7.3a (AS 1.10.22)
Yes, 6 Enhanced enhancers, 18 blocks distance.
Enhancers attunement match the ritual, all are 900/100/100, Lucerna.
Tried with and without the Ritual Anchor.
3x3 area centered on ritual is chunkloaded (ExU2 Chunk Loading Ward)

![2020-05-14_09 00 42](https://user-images.githubusercontent.com/808397/81903404-03502700-95c2-11ea-9a93-c7e45ac76561.png)
![2020-05-14_09 00 48](https://user-images.githubusercontent.com/808397/81903408-0519ea80-95c2-11ea-9129-5a16b199175b.png)

**Comment by Doomgull** [comment]
Please replicate this without sponge.

**Comment by NopeDK** [comment]
I have now confirmed that it is still bugged after removing Sponge.

**Comment by faraday326** [comment]
This is replicating for me on a server running the most recent FTB:I (2.0.9), astral version (1.10.25). 

On every server restart, all four of my rituals stop working (2 vicio using an anchor, and 2 lucerna not using an anchor). 

At least one other person playing on the same server is _not_ experiencing the issue so it's not a server-wide issue. 

I've tried both 1) chunkloading the entire ritual, 2) chunkloading none of the ritual, and 3) making sure the rituals are in a single chunk. None of these seem to help.

**Comment by JonasDoe** [comment]
I know, the issue is closed, but I'm facing the same issue in SevTech: Ages 3.2.3 with astralsorcery-1.12.2-1.10.21. It wasn't alway like that, though. The first days it was fine, but now I've gotta rebuild all the altars everytime the server has started.

**Comment by PocketfulOfGeese** [comment]
Completely unrelated issue to do with caching weirdness that happens with sevtech.

Also 1.12 is not bug supported anymore by this mod, and hasn't been for well over a year.
